examples/parents/examples-bom/pom.xml

```diff
@@ -81,18 +81,18 @@
             </dependency>
             <dependency>
                 <groupId>org.xmlunit</groupId>
-                <artifactId>xmlunit-core</artifactId>
-                <version>2.7.0</version>
+                <artifactId>xmlunit-legacy</artifactId>
```

Comment:
We are actually moving the other way, so things should be migrated to use just xmlunit-core. See:
- https://github.com/xmlunit/user-guide/wiki/Migrating-from-XMLUnit-1.x-to-2.x
- https://git.opendaylight.org/gerrit/c/netconf/+/105686